controllers/articles.controllers.js

```diff
@@ -0,0 +1,31 @@
+const {readArticles, readArticlesById} = require('../models/articles.models');
+
+const getArticles = (request, response,next) => {
+
+    const {article_id} = request.params;
```

Comment:
Looks like bits of a previous task have slipped in here. Please try to keep each task on its own branch for easier reading